common/app/views/fragments/contentAgeNotice.scala.html

```diff
@@ -1,8 +1,10 @@
 @(ageMessage: String, isHidden: Boolean = false, ariaHidden: Boolean = false)(implicit request: RequestHeader)
 
 @if(ageMessage.nonEmpty) {
-    <div class="old-article-message @if(isHidden){u-h}" @if(ariaHidden){aria-hidden="true"}>
-        @fragments.inlineSvg("clock", "icon", List("old-article-message--clock"))
-        This article is more than <strong>@ageMessage old</strong>
+    <div class="old-article-message-wrapper @if(isHidden){u-h}" @if(ariaHidden){aria-hidden="true"}>
```

Comment:
Just for my own curiosity: What do `isHidden` and `ariaHidden` do here? why would you want to hide this element separately from sighted users/screen readers? /cc @SiAdcock @guardian/guardian-frontend-team

Reply:
I added them so we can place the notice for SIGHTED users above the headline. But for people using screen readers it's below the headline - as they often navigate directly to headlines. (Simon taught me this)